# Request - Remove an object from the planning scene.
# A unique identifier for the object, or 'all' to remove all objects (except for the robot) from the scene
string object_identifier
---
# Result
# Bool that indicates whether the object was successfully removed
bool success
